ShopAppTests/App/Account/Controllers/AddressFormViewControllerSpec.swift

```diff
@@ -0,0 +1,277 @@
+                zipCodeTextFieldView = self.findView(withAccessibilityLabel: "addressFormZipCode", in: viewController.view) as! InputTextFieldView
+                phoneTextFieldView = self.findView(withAccessibilityLabel: "addressFormPhone", in: viewController.view) as! InputTextFieldView
+                submitButton = self.findView(withAccessibilityLabel: "addressFormSubmitButton", in: viewController.view) as! BlackButton
+            }
+            
+            it("should have a correct view model type") {
+                expect(viewController.viewModel).to(beAKindOf(AddressFormViewModel.self))
+            }
+            
+            it("should have pickers with correct placeholders") {
+                expect(countryPicker.placeholder) == "Placeholder.Country".localizable.required.uppercased()
+                expect(statePicker.placeholder) == "Placeholder.State".localizable.uppercased()
+            }
+            
+            it("should have text field views with correct placeholders") {
+                expect(nameTextFieldView.placeholder) == "Placeholder.Name".localizable.required.uppercased()
+                expect(lastNameTextFieldView.placeholder) == "Placeholder.LastName".localizable.required.uppercased()
+                expect(addressTextFieldView.placeholder) == "Placeholder.Address".localizable.required.uppercased()
+                expect(addressOptionalTextFieldView.placeholder) == "Placeholder.AddressOptional".localizable.uppercased()
+                expect(cityTextFieldView.placeholder) == "Placeholder.City".localizable.required.uppercased()
+                expect(zipCodeTextFieldView.placeholder) == "Placeholder.ZipCode".localizable.required.uppercased()
+                expect(phoneTextFieldView.placeholder) == "Placeholder.PhoneNumber".localizable.required.uppercased()
+            }
+            
+            it("should have button with correct title") {
+                expect(submitButton.title(for: .normal)) == "Button.Submit".localizable.uppercased()
+            }
+        }
+        
+        describe("when data loaded") {
+            context("if data is empty") {
+                beforeEach {
+                    viewController = UIStoryboard(name: StoryboardNames.account, bundle: nil).instantiateViewController(withIdentifier: ControllerIdentifiers.addressForm) as! AddressFormViewController
+                    
+                    let repositoryMock = PaymentsRepositoryMock()
+                    let addressFormUseCaseMock = CountriesUseCase(repository: repositoryMock)
+                    viewModelMock = AddressFormViewModelMock(countriesUseCase: addressFormUseCaseMock)
+                    viewController.viewModel = viewModelMock
+                    
+                    countryPicker = self.findView(withAccessibilityLabel: "addressFormCountryPicker", in: viewController.view) as! BasePicker
```

Comment:
Can we avoid this init in each test case? We already do it in previous beforeEach method